src/vaults/Escrow.sol

```diff
@@ -57,6 +57,22 @@ contract Escrow is Auth, IEscrow {
             emit Approve(asset, tokenId, spender, 0);
```

Comment:
I believe we should remove all `approve` related methods from `Escrow` now?

Reply:
Good point

Reply:
We can't the router escrow needs approval